app-doc/kicad-doc/kicad-doc-4.0.6.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils
+
+DESCRIPTION="Electronic Schematic and PCB design tools manuals"
+HOMEPAGE="http://www.kicad-pcb.org/"
+SRC_URI="https://github.com/KiCad/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="|| ( GPL-3+ CC-BY-3.0 ) GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="html +pdf"
+LANGS="en fr it ja nl pl"
+for lang in ${LANGS}; do
+	LUSE+=" l10n_${lang}"
+done
+IUSE+=${LUSE}
+REQUIRED_USE="|| ( html pdf ) ^^ ( ${LUSE} )"
+unset lang
+unset LUSE
+
+DEPEND="
+	>=app-text/asciidoc-8.6.9
+	>=app-text/dblatex-0.3.10
+	>=app-text/po4a-0.45
+	>=sys-devel/gettext-0.18
+	dev-perl/Unicode-LineBreak
+	dev-util/source-highlight
+	l10n_en? ( dev-texlive/texlive-langenglish )
+	l10n_fr? ( dev-texlive/texlive-langfrench )
+	l10n_it? ( dev-texlive/texlive-langitalian )
+	l10n_nl? ( dev-texlive/texlive-langeuropean )
+	l10n_pl? ( dev-texlive/texlive-langpolish )
+	l10n_ja? ( dev-texlive/texlive-langjapanese media-fonts/vlgothic )"
+RDEPEND=""
+
+src_prepare() {
+	sed -e "s|KICAD_DOC_INSTALL_PATH share/doc/kicad|&-${PV}|g" -i CMakeLists.txt || die "sed failed"
```

Comment:
This does not seem to work:

```
./usr/share/doc/kicad/help/
./usr/share/doc/kicad/help/pl/
./usr/share/doc/kicad/help/pl/pl_editor.pdf
./usr/share/doc/kicad/help/pl/pcbnew.pdf
```

Besides, it should be `PF`, not `PV`.

Reply:
I've debugged this issue and was surprised at what I found:
- sed works as expected (yeah, could have been `PF` though, but whatever)
- KICAD_DOC_INSTALL_PATH is completely ignored in the build file, it seems they intended to use it but never did. The hard coded install path is /usr/share/doc/kicad/help/{LANG}
- Technically I could patch the CMakeList file in the CMakeModules, however that would still not help much, since kicad itself looks for the help files in the same hard coded location (you can see that in the kicad source code in the file `common/searchhelpfilefullpath.cpp`.

So I've decided to the the following:
- remove the src_prepare section, since it's useless
- open bugs to upstream about the issue

In the mean time we will have to put up with the non Gentoo path for the docs otherwise it breaks help for kicad users.